db/model/aspect.js

```diff
@@ -179,21 +177,27 @@ module.exports = function aspect(seq, dataTypes) {
     },
     hooks: {
 
+      /**
+       * TODO:
```

Comment:
"TODO"? Is there user story for this? Or are you doing this in subsequent PR for this same story?

Reply:
I will be creating stories to address the TODOs.